libs/shared/shared/django_apps/reports/migrations/0043_commitreport_reports_commitreport_commit_code_unique.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 4.2.21 on 2025-09-08 14:48
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('reports', '0042_alter_testflagbridge_id'),
+    ]
+
+    operations = [
+        migrations.RiskyAddConstraint(
+            model_name='commitreport',
```

Comment:
Oh, I think you need the import for the Risky Models. 

```suggestion
from shared.django_apps.migration_utils import RiskyAddConstraint

from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('reports', '0042_alter_testflagbridge_id'),
    ]

    operations = [
        RiskyAddConstraint(
            model_name='commitreport',
```